.rubocop.yml

```diff
@@ -6,3 +6,7 @@ AllCops:
   Exclude:
     - 'spec/internal/**/*'
     - 'spec/test_app_templates/**/*'
+
+Metrics/ClassLength:
+  Exclude:
+    - 'lib/spotlight/dor/indexer.rb'
```

Comment:
If we don't like this soln, is there any value in lines:  https://github.com/sul-dlss/spotlight-dor-resources/blob/master/lib/spotlight/dor/indexer.rb#L42-L44